docs/drivers.md

```diff
@@ -31,21 +104,16 @@ This will add the bit-node package as a dependency.
 
 ## Using JavaScript Components
 
-To use Bit components within your JS code you need to first require bit-node:
-
-```js
-const bit = require('bit-js');
-```
-
-Now it's a matter of calling the right components, like so:
+To use Bit components within your JS code you simply need to require them (after importing them, of course)
 
 ```js
-isString = bit('is-string');
-console.log(isString('Hello World');
+const isString = require('bit/is-string');
 ```
 
 That's it :)
 
-### Runtime Dependency Resolution
+## Using JavaScript driver in a build process
+
+If you are using Bit to manage your code components, the components are not a part of the codebase. This means that in order to populate the `components` directory (and all the links in the `node_modules`), you will need to use bitjs.
 
-// TODO
\ No newline at end of file
+Bit-javascripts is able to both fetch components, and link them properly using a single command - `bitjs import`. This means that you can simply need to add Bit-js in your `package.json` file for your project, and a post-install hook to run `bitjs import`. This way, after installing all the packages (including bit-js), bit-js will be able to bring all components and link them, so your code will be able to require them.
```

Comment:
> Bit-javascripts is able to both fetch components, and link them properly using a single command - `bitjs import`. This means that you can simply need to add Bit-js in your `package.json` file for your project, and a post-install hook to run `bitjs import`. This way, after installing all the packages (including bit-js), bit-js will be able to bring all components and link them, so your code will be able to require them.

**bit-javascript** can both fetch components and link them using the command - `bitjs import`. If you want to automatically install the bit components dependencies after you call `npm install` or `yarn` **you can add bit-javascript** to your `package.json` file, and configure a post-install hook to run `bitjs import`. This way, after installing all the packages, **bit-javascript** will be able to **import all of the** components and link them.